go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/OpenEugene/openboard
+
+go 1.12
+
+require (
+	github.com/OpenEugene/openboard/back/internal v0.0.0-20191127014750-422d8d11bc01
+	golang.org/x/tools v0.0.0-20190524140312-2c0ae7006135 // indirect
+	google.golang.org/grpc v1.25.1
+	honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc // indirect
+)
```

Comment:
This file is unnecessary. Either a go.mod needs to be added to a relevant subdir, or the existing go.mod files need to be updated. We can clarify this at the next meeting.

Reply:
I'll remove this file.